src/tags-input.less

```diff
@@ -1,5 +1,6 @@
 .tags-input {
-	display: inline-block;
+	display: inline-flex;
+	flex-wrap: wrap;
```

Comment:
I would say `min-width` should be `25%` since the old calculation was `Math.max(..., w / 4)`.
And maybe also add a `min-width` on the whole `.tags-input`.
Btw thank you guys, this component is awesome :smile: :+1: